lib/al/Library/Layout/LayoutActorUtil.h

```diff
@@ -1,19 +1,121 @@
+bool isContainPointPane(const al::IUseLayout*, const char*, const sead::Vector2f&);
+void findHitPaneFromLayoutPos(const al::IUseLayout*, const sead::Vector2f&);
+bool isExistHitPaneFromLayoutPos(const al::IUseLayout*, const sead::Vector2f&);
+void findHitPaneFromScreenPos(const al::IUseLayout*, const sead::Vector2f&);
+bool isExistHitPaneFromScreenPos(const al::IUseLayout*, const sead::Vector2f&);
+bool isTouchPosInPane(const al::IUseLayout*, const char*);
+void setCursorPanePos(al::IUseLayout*, const al::IUseLayout*);
+void setPaneVtxColor(const al::IUseLayout*, const char*, const sead::Color4u8&);
+bool isTriggerTouchPane(const al::IUseLayout*, const char*);
+bool isHoldTouchPane(const al::IUseLayout*, const char*);
+bool isReleaseTouchPane(const al::IUseLayout*, const char*);
+s32 getPaneChildNum(const al::IUseLayout*, const char*);
+const char* getPaneChildName(const al::IUseLayout*, const char*, s32);
+void setPaneStringLength(al::IUseLayout*, const char*, const char16_t*, u16, u16);
+void setPaneString(al::IUseLayout*, const char*, const char16_t*, u16);
+void setPaneCounterDigit1(al::IUseLayout*, const char*, s32, u16);
+void setPaneCounterDigit2(al::IUseLayout*, const char*, s32, u16);
+void setPaneCounterDigit3(al::IUseLayout*, const char*, s32, u16);
+void setPaneCounterDigit4(al::IUseLayout*, const char*, s32, u16);
+void setPaneCounterDigit5(al::IUseLayout*, const char*, s32, u16);
+void setPaneCounterDigit6(al::IUseLayout*, const char*, s32, u16);
+void setPaneNumberDigit1(al::IUseLayout*, const char*, s32, u16);
+void setPaneNumberDigit2(al::IUseLayout*, const char*, s32, u16);
+void setPaneNumberDigit3(al::IUseLayout*, const char*, s32, u16);
+void setPaneNumberDigit4(al::IUseLayout*, const char*, s32, u16);
+void setPaneNumberDigit5(al::IUseLayout*, const char*, s32, u16);
+void setPaneStringFormat(al::IUseLayout*, const char*, const char*, ...);
+void setTextPositionCenterH(al::IUseLayout*, const char*);
+void initPaneMessage(al::IUseLayout*, const char*, const al::MessageHolder*, const char*, u32);
+void setPaneSystemMessage(al::LayoutActor*, const char*, const char*, const char*);
+void setPaneStageMessage(al::LayoutActor*, const char*, const char*, const char*);
+const char16_t* getPaneStringBuffer(const al::IUseLayout*, const char*);
+s32 getPaneStringBufferLength(const al::IUseLayout*, const char*);
+void setTextBoxPaneFont(const al::LayoutActor*, const char*, const char*);
+void adjustPaneSizeToTextSizeAll(const al::LayoutActor*);
+void requestCaptureRecursive(const al::LayoutActor*);
+void setRubyScale(const al::LayoutActor*, float);
+nn::ui2d::TextureInfo* createTextureInfo();
+nn::ui2d::TextureInfo* createTextureInfo(const agl::TextureData&);
+nn::ui2d::TextureInfo* createTextureInfo(const al::IUseLayout*, const char*);
+void getPaneTextureInfo(nn::ui2d::TextureInfo*, const al::IUseLayout*, const char*);
+nn::ui2d::TextureInfo* createTextureInfo(const char*, const char*, const char*);
```

Comment:
wrong return type: Should be custom `'anonymous namespace'::ResourceTextureInfo`